crates/sargon/src/system/sargon_os/sargon_os.rs

```diff
@@ -686,8 +720,11 @@ mod tests {
     #[actix_rt::test]
     async fn test_delete_wallet() {
         let test_drivers = Drivers::test();
-        let bios = Bios::new(test_drivers);
-        let os = SUT::boot(bios).await;
+        let clients = Clients::new(Bios::new(test_drivers));
```

Comment:
DRY: I would perhaps update these tests to use another `ctor` than `boot`, maybe `boot_with_test_interactors`...? and then you don't need to repeat those lines being repeated?

Reply:
I would love to, but we have so many many tests already written like so, this would constitute to its own PR.